hr_employee_document/models/hr_employee.py

```diff
@@ -7,30 +7,23 @@
 class HrEmployee(models.Model):
     _inherit = "hr.employee"
 
-    document_ids = fields.One2many(
-        "ir.attachment",
-        compute="_compute_document_ids",
-        groups="hr.group_hr_user",
-        string="Documents",
-    )
     documents_count = fields.Integer(
```

Comment:
Please avoid this plural:
```suggestion
    document_count = fields.Integer(
```